index.test.js

```diff
@@ -330,6 +332,42 @@ describe('TestSummary', () => {
   });
 });
 
+describe('action logic', () => {
+  // Setup the necessary mocks
+  const token = 'dummyToken';
+  const inputPath = '**/TEST-*.xml';
+  const includeSummary = true;
+  const numFailures = 10;
+  const name = 'Junit Results';
+  const repo = 'foo/bar';
+  const ref = 'refs/heads/some-ref';
+  const sha = '1234567890123456789012345678901234567890';
+
+  beforeAll(() => {
+    process.env['INPUT_ACCESS-TOKEN'] = token;
+    process.env['INPUT_PATH'] = inputPath;
+    process.env['INPUT_INCLUDESUMMARY'] = includeSummary;
+    process.env['INPUT_NUMFAILURES'] = numFailures;
+    process.env['INPUT_NAME'] = name;
+    process.env['GITHUB_REPOSITORY'] = repo;
+    process.env['GITHUB_REF'] = ref;
+    process.env['GITHUB_SHA'] = sha;
+  });
+
+  it('should provide "summary" output', async done => {
+    const ip = path.join(__dirname, 'index.js');
+    cp.exec(`node ${ip}`, {env: process.env}, (error, stdout, stderr) => {
+      try {
+        console.log(`STDOUT: ${stdout}`);
```

Comment:
Do you need to log the stdout here ? In case of failure, the assertion should already give a helpful message to identify the root cause.

Reply:
No, that was just an oversight. Will correct.